test/src/unit-serialization.cpp

```diff
@@ -173,4 +173,19 @@ TEST_CASE("serialization")
             test("\xE1\x80\xE2\xF0\x91\x92\xF1\xBF\x41", "\\ufffd" "\\ufffd" "\\ufffd" "\\ufffd" "\x41");
         }
     }
+
+    SECTION("to_string")
+    {
+        auto test = [&](std::string const & input, std::string const & expected)
+        {
+            using std::to_string;
```

Comment:
You should not have to use that for that specific test. The `using std::thing` is only necessary in generic code.